count-vowels/starter.py

```diff
@@ -0,0 +1 @@
+def count-vowels(text):
```

Comment:
Don't put dashed in function names in python (notice how the syntax highlighting looks weird on "vowels"